pkg/console/util.go

```diff
@@ -120,12 +125,12 @@ func customizeConfig() {
 			Content:            getHarvesterManifestContent(harvesterChartValues),
 		},
 	}
+	cfg.Config.K3OS.Labels["svccontroller.k3s.cattle.io/enablelb"] = "true"
```

Comment:
The klipper LB only provides access to one of the node IP so we might need to change it for HA. In that context, this label is not needed then.